external/header.php

```diff
@@ -42,8 +55,11 @@ public static function __callstatic($name, $arguments)
     header("Location: $newURI");
     exit;
   }
-
-  if (isset($_COOKIE['_profile']) && $_COOKIE['_profile'] || PHP_SAPI == 'cli' && ((isset($_SERVER['XHPROF_PROFILE']) && $_SERVER['XHPROF_PROFILE']) || (isset($_ENV['XHPROF_PROFILE']) && $_ENV['XHPROF_PROFILE'])))
+  
+  if (isset($_COOKIE['_profile']) && $_COOKIE['_profile'] || PHP_SAPI == 'cli' && ( (isset($_SERVER[strtoupper($_xhprof['profiler']).'_PROFILE'])  
```

Comment:
1. Extract `$_xhprof['profiler']).'_PROFILE'` into temp variable (e.g. `$envVariableName`) in this large IF statement to increase code readability.
2. Reformat that IF statement in a way that each OR condition would be on it's own line starting with `|| ` will help a lot.